apps/etterlatte-beregning/src/main/kotlin/beregning/BeregnBarnepensjonService.kt

```diff
@@ -254,15 +256,19 @@ class BeregnBarnepensjonService(
                 )
             }
         },
-        institusjonsopphold = PeriodisertBeregningGrunnlag.lagGrunnlagMedDefaultUtenforPerioder(
-            beregningsGrunnlag.institusjonsoppholdBeregningsgrunnlag?.mapVerdier { institusjonsopphold ->
-                FaktumNode(
-                    verdi = institusjonsopphold,
-                    kilde = beregningsGrunnlag.kilde,
-                    beskrivelse = "Institusjonsopphold"
-                )
-            } ?: listOf()
-        ) { _, _, _ -> FaktumNode(null, beregningsGrunnlag.kilde, "Institusjonsopphold") }
+        institusjonsopphold = if (featureToggleService.isEnabled(BrukInstitusjonsoppholdIBeregning, false)) {
```

Comment:
Må finne på noko meir fornuftig på sikt, men for at det her ikkje lengre skal stoppe i prod